### ISSUE TITLE
Hiding/Showing data changes the viewport position

### ISSUE THREAD

**Issue by crazydef** [awulkiew/graphical-debugging#33]
I'm viewing several lists of points in the Geometry Watch window and I've noticed when I show or hide different variables the viewport position is jumping around.  Presumably to recentre itself on the remaining points, but that's monumentally unhelpful when I'm just trying to see what the differences are in the region I'm focused on.

Hiding and showing data should not adjust either the position or the scale of the viewport.  There should be separate controls for that.

**Comment by awulkiew** [comment]
Yes, it is "jumping" because the view is always relative to the visible geometries (their bounding box). I agree that this should be changed and e.g. done only if there is no zoom. Thanks!
